ChatTTS/model/gpt.py

```diff
@@ -138,6 +138,44 @@ def prepare(self, compile=False):
             except RuntimeError as e:
                 self.logger.warning(f"compile failed: {e}. fallback to normal mode.")
 
+    def __call__(
```

Comment:
No, don't remain this code. It has been moved to embed.py.

Reply:
ok